client/src/App.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import index from './index.jsx';
+
+it('renders without crashing', () => {
```

Comment:
This is a great initial start to your tests! Using `enzyme` or `react-testing-library` are great natural next steps to explore.